utils/build-script-impl

```diff
@@ -2221,6 +2236,22 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DFoundation_DIR:PATH=$(build_directory ${host} foundation)/cmake/modules
                 )
 
+                if [[ $(is_cross_tools_host ${host}) ]] ; then
+                    cmake_options+=("${SWIFT_TARGET_CMAKE_OPTIONS[@]}")
+
+                    # llbuild looks for the SQlite3 library using find_package(),
+                    # so search for it in CROSS_COMPILE_DEPS_PATH using the CMake
+                    # process for doing so and don't use cross-compiled
+                    # executables, see the linked CMake docs for more info:
+                    #
+                    # https://cmake.org/cmake/help/latest/command/find_package.html
+                    # https://cmake.org/cmake/help/latest/command/find_program.html
```

Comment:
Great comment! This is exactly what I wanted!

I think you put in the link twice. But feel free to fix in a subsequent commit.

Reply:
If you mean the two links here, it is because llbuild calls both `find_package` and `find_program`, which is why I had to tell it not to look in the cross-compilation SDK for programs to run in the host, with `CMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER`.